components/faucet/CTA.tsx

```diff
@@ -0,0 +1,49 @@
+import { Flex } from "@chakra-ui/react";
+import Link from "next/link";
+import type { FC } from "react";
+import { Button, Text } from "tw-components";
+
+const CTA: FC = () => {
+  return (
+    <Flex
+      bg="url(/assets/faucet/cta-bg.png) no-repeat center"
+      bgSize="cover"
```

Comment:
I think you can achieve this entire piece better by using our `<Card>` component (and then just giving it the bg image)